public/demo.js

```diff
@@ -173,8 +173,8 @@ $(document).ready(function() {
             return false;
         }
 
-        if (($("#dropdownMenuInput").text()).toLowerCase().indexOf("language") < 0) {
-            var source = getLanguageCode($.trim($("#dropdownMenuInput").text()));
+        if (getSrcLang().toLowerCase().indexOf("language") < 0) {
```

Comment:
the instances and memory are different for the livedemos because they have
more traffic

On Mon, Jan 4, 2016, 4:37 PM Nathan Friedly <notifications@github.com>
wrote:

> In public/demo.js
> <https://github.com/watson-developer-cloud/language-translation-nodejs/pull/15#discussion_r48770333>
> :
>
> > @@ -173,8 +173,8 @@ $(document).ready(function() {
> >              return false;
> >          }
> >
> > -        if (($("#dropdownMenuInput").text()).toLowerCase().indexOf("language") < 0) {
> > -            var source = getLanguageCode($.trim($("#dropdownMenuInput").text()));
> > +        if (getSrcLang().toLowerCase().indexOf("language") < 0) {
>
> Ok, then we can probably remove that if statement.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/watson-developer-cloud/language-translation-nodejs/pull/15/files#r48770333>
> .
>